### PR TITLE
Updated to latest laravel version

### PR DESCRIPTION
**Pull request by cpickhardt15**

### CODE REVIEW

**Comment by joselfonseca** [comment]
@cpickhardt15 Hi there, could you please remove support for PHP 7.1 in the travis configuration so we can merge it. 

Thanks!

**Comment by cpickhardt15** [comment]
Hope that helps! Slight confusion, but hopefully this is what you meant. @joselfonseca

File: composer.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
         }
     ],
     "require": {
-        "php": "^7.1.3",
+        "php": "^7.2",
         "dingo/api": "^2.3",
```
Comment (kslimani) [inline composer.json]: If the goal is to upgrade to Laravel 6, i suggest to change laravel/framework to "^6.0", but also dingo/api to "^2.4", laravel/passport to "^7.5", spatie/laravel-permission to "^3.0"

Comment (kslimani) [reply composer.json]: Also, beyondcode/laravel-dump-server is deprecated for facade/ignition "^1.4"
Finally, i also suggest phpunit/phpunit  to "^8.0"

Comment (joselfonseca) [reply composer.json]: I agree, At first i thought it was just a composer update, but if this is for the issue https://github.com/joselfonseca/laravel-api/issues/47 then the actual update is to Laravel 6 which means updating all the packages and updating the actual app directory and config files.

**Comment by joselfonseca** [comment]
@cpickhardt15 I've changed the base to the 7.0 branch since that is the one this PR will apply, the master branch now is on laravele 6. thanks!

**Pull request merged**